test/powershell/Language/Scripting/DeserializedMethods.Tests.ps1

```diff
@@ -0,0 +1,27 @@
+﻿Describe "DeserializedMethods" -Tags "CI" {    
+    It "Deserialized objects shouldn't ever have any methods (unless they are primitive known types)" {
+        $a = new-object collections.arraylist
```

Comment:
fixed

---
In reply to: [78832342](https://github.com/Powershell/Powershell/pull/2118#discussion_r78832342) [](ancestors = 78832342)